index.html

```diff
@@ -723,7 +726,7 @@ <h2>Terms of Use</h2>
     "ageOver": 21
   },
   <span class="highlight">"termsOfUse": [{
-    "type": "Policy",
+    "type": "< term of use type ID >",
```

Comment:
Instead of using placeholder syntax here and below just put in an example value.  These are just examples and everything else has example values.

Reply:
Done in #170